scripts/update_schemas_version.sh

```diff
@@ -0,0 +1,8 @@
+# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+    grep '"tag_name":' |                                            # Get tag line
+    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
+}
+
+get_latest_release "onsdigital/eq-questionnaire-schemas" > .schemas-version
```

Comment:
Now that this is a shell script we should add the relevant lines to identify it:
```
#!/usr/bin/env bash
```
And to exit immediately on failure:
```
set -e
```

This is used in a number of our other scripts in this repo.

Reply:
Done, thanks! 👍